src/styles/_base_lists.scss

```diff
@@ -8,28 +8,7 @@ li {
   padding: 0;
 }
 
-[class*="__list"],
-[class*="__menu"],
-[class*="__nav"] {
+ul,
+li {
```

Comment:
Should not be `ul, ol` here?
`<li>` elements don't need a `list-style` property.

Reply:
Hot-diggity-dog, good catch!